src/messages/imageloadermanager.cpp

```diff
@@ -80,9 +83,10 @@ void ImageLoaderWorker::handleLoad(chatterino::messages::LazyLoadedImage *lli, Q
     }
 
     lli->emoteManager.incGeneration();
-    lli->windowManager.layoutVisibleChatWidgets();
+    //lli->windowManager.layoutVisibleChatWidgets();
```

Comment:
Just remove this line, it's useless to leave it commented out when we've moved it elsewhere.